CMakeLists.txt

```diff
@@ -167,3 +160,24 @@ function(mbed_generate_bin_hex target)
         VERBATIM
     )
 endfunction()
+
+#
+# Configure a Mbed OS application.
+#
+function(mbed_configure_application target)
+    mbed_set_language_standard(${app_target})
+    mbed_set_mbed_target_linker_script(${app_target})
+    mbed_generate_bin_hex(${app_target})
+endfunction()
+
+#
+# Configure Mbed OS CMake component (default).
```

Comment:
```suggestion
# Configure Mbed OS component.
```

What did you mean by "`(default)`"?

Reply:
That this is default one provided by Mbed but can be overwritten (something else used). I can remove default or should we rephrase?

Reply:
> That this is default one provided...

The default what?